src/packages/client/jest.config.js

```diff
@@ -14,6 +14,7 @@ module.exports = {
     'index.d.ts',
     'index.js',
     'index.test-d.ts',
+    '.bench.ts',
```

Comment:
```suggestion
    '*.bench.ts',
```
Probably needs a *

Reply:
Patterns are from https://github.com/micromatch/micromatch

Reply:
I thought so too... But it raised [this](https://github.com/prisma/prisma/runs/1777481598#step:7:535)

Reply:
ok we'll see then 🤷